tests/console/repo_package_install.pm

```diff
@@ -14,23 +14,53 @@ use base "consoletest";
 use strict;
 use testapi;
 use utils;
-use version_utils qw(sle_version_at_least is_jeos);
+use version_utils qw(is_sle is_jeos);
 
 my %packages = (
-    # On JeOS Salt is present in the default image
     salt => {
         repo      => 'Basesystem',
-        installed => is_jeos() ? 1 : 0
-    });
+        installed => is_jeos() ? 1 : 0,       # On JeOS Salt is present in the default image
+        condition => sub { is_sle('15+') },
+    },
+    'update-test-feature' => {                # See poo#36451
+        repo      => 'SLES12',
+        installed => 0,
+        available => sub { get_var('BETA') },
+        condition => sub { is_sle('<15') },
```

Comment:
If you tell me in which repo this package will be and if it will be available at all in default SLES installation, we can adjust expected repo and remove condition. As of now it will work for SLE 12 SP4 only and we will keep ticket to enable it for SLE15.

Not sure what you mean, it says package not found, and is not expected to be installed. So it can be the case that package is there, but is installed. If we don't want to check this part, that's another point.

Reply:
Yes, I am not sure if it will be basesystem repo. If you are, we can put expectations for SLE 15 too.

Reply:
So, do we have an agreement here? I still do not get if you propose putting expectations for SLE 15 or you are fine with handling it when we have SLE 15?

Reply:
I've confirmed with Frederic, that these packages should be available in Basesystem repo for SLE15, so adjusted expectations accordingly. As of now we don't have any build of SLE15 to have this test working.